### PR TITLE
Data Cleansing Admin Settings

### LINKED ISSUE #583: Admin screen to control Data Cleansing settings
We will develop a much nicer spec for this feature, but here is a rudimentary beginning for what might be controlled on this screen. 
image

### PR DESCRIPTION
**Pull request by axelstudios**
image

### Details
This pull request includes database migrations for storing data cleansing rules, the rule admin page, get/set/reset ruleset functionality, and selective application of rules after mapping.

### Update commands
```bash
./manage.py migrate
```

### Recommended tests to attempt before merging:
- Run API tests (completed by CI)
- Run Cleansing tests (completed by CI)
- Check permissions (only organization owners can see/change rules, sub-orgs should use parent org's rules)

### Possible future features:
- `Present date` selection
- Input sanity checking (if no min or max don't allow saving, check that max is greater than min, etc)
- Possible confirmation dialog before restoring defaults
- Display the Missing Matching Field and Missing Values rules, but don't allow editing
- See `Out of scope` section: https://docs.google.com/document/d/1U5t04k_-My8GwzCSD721d8Rs7M_x4chAdfiGANUimGQ/edit?usp=sharing

Closes #583

### CODE REVIEW

**Comment by nllong** [comment]
Can you fix the 3 failing tests? It looks like they are failing because of the change from this being a database driven model instead of the JSON backed model.

**Comment by nicholasserra** [comment]
Couple of linting issues too. One will be fixed when you merge develop in.

File: seed/cleansing/models.py [hunk 1]
```diff
@@ -16,34 +15,271 @@
...
+                org=organization,
+                field=rule['field'],
+                category=CATEGORY_IN_RANGE_CHECKING,
+                type=rule['type'],
+                min=rule.get('min'),
+                max=rule.get('max'),
+                severity=rule['severity'],
+                units=rule.get('units', '')
+            )
+        # for pair in data_type_check:
```
Comment (nicholasserra) [inline seed/cleansing/models.py]: Noting commented out code

**Comment by RDmitchell** [comment]
@nllong, @axelstudios, @mmclark: Looks like all the tests have passed, so can we label this "ready" ?? I am thinking yes?

**Comment by axelstudios** [comment]
@RDmitchell I believe so - @nllong tested via Docker and said everything looked good.

**Comment by nllong** [comment]
yep, this worked on a new install as expected. I think it is safe to merge.

**Pull request merged**